src/util.ts

```diff
@@ -34,14 +36,14 @@ export interface IMakeUploadFileData {
 }
 
 export class Util {
-	// tslint:disable-next-line no-any
-	public static async DownloadFile(url: string, options: any = {}): Promise<Buffer> {
+	// tslint:disable-next-line:max-line-length
+	public static async DownloadFile<T extends Response | Response["body"]>(url: string, options: OptionsOfDefaultResponseBody = {}): Promise<Buffer> {
 		if (!options.method) {
 			options.method = "GET";
 		}
 		options.url = url;
-		options.encoding = null;
-		return await request(options);
+		options.encoding = undefined;
+		return got(options).buffer();
```

Comment:
`return await got(options).buffer();`, else it is not clear where errors are thrown, if they happen

Reply:
👍